lib/tasks/reindex_best_bets.rake

```diff
@@ -0,0 +1,23 @@
+desc "Resend all local queries to rummager to be reindexed"
+task reindex_best_bets: :environment do
+  message = <<-MSG
+    Rebuilding the elasticsearch index will just resend all locally stored best bets across to
+    be insertedin to elasticsearch. This means that any orphanded entried in the elasticsearch
+    index would still exist. This can be avoided by running the below rake task on Rummage before
+    rebuilding the index.
+
+    bundle exec rake rummager:switch_to_empty_index RUMMAGER_INDEX=metasearch
```

Comment:
I think it's worth drawing attention to here as well, since someone might just read this file and miss it.

Out of interest, do we know how rummager behaves when the metasearch index is empty? Will it fall back to querying without best bets?